packages/neon-js/src/experimental/helpers.ts

```diff
@@ -208,3 +232,55 @@ export async function addFees(
     );
   }
 }
+
+/**
+ * Deploy a smart contract
+ * @param NEF - A smart contract in Neo executable file format. Commonly created by a NEO compiler and stored as .NEF on disk
+ * @param manifest - the manifest conresponding to the smart contract
+ * @param config -
+ */
+export async function deployContract(
+  NEF: Buffer,
```

Comment:
Buffer isnt a common data format for neon-js. Try relaxing this constraint by allowing the web variant which is [ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer). Probably can do Buffer | ArrayBuffer.

Reply:
Might be good to look if my way of checking if the input is an ArrayBuffer type is correct.